lib/Echidna/Types/Tx.hs

```diff
@@ -55,6 +57,18 @@ data Tx = Tx { _call  :: TxCall       -- | Call
 makeLenses ''Tx
 $(deriveJSON defaultOptions ''Tx)
 
+basicTx :: Text -> [AbiValue] -> Addr -> Addr -> Word -> Tx
+basicTx f a s d g = basicTxWithValue f a s d g 0
```

Comment:
could we provide more self-explanatory names? or some docs, or type aliases?

Reply:
i guess we could change it to `contractCall` and `create` or something, do you have any suggestions?

Reply:
I think `basicTx` is fine. It's the `f`, `a`, `s`, `d`, `g` vars and their types which is hard to decipher and to use the function I need to look at the Tx definition. A doc for `basicTx` describing the args should do the job.